runtime/doc/lua.txt

```diff
@@ -1378,6 +1378,8 @@ notify_once({msg}, {level}, {opts})                        *vim.notify_once()*
                              |vim.log.levels|.
                     {opts}   (table|nil) Optional parameters. Unused by
                              default.
+                    {key}    (string|nil) Optional key unique to the message.
+                             If not given, {msg} is used.
```

Comment:
seems like a good time to start using the `opts` thing...

Reply:
`opts` is not used in `vim.notify` at all currently, so how about just handling the behavior of `vim.notify_once` inside `vim.notify` then? If `opts.once` is `true` then it behaves like `notify_once`, and with a string it will use that string as the unique identifier for the message.

Reply:
One thing to note, I believe some plugins are replacing `vim.notify` to show messages in floating windows and stuff like that, so they would have to add that behavior too.

Reply:
> so how about just handling the behavior of `vim.notify_once` inside `vim.notify` then? If `opts.once` is `true` then it behaves like `notify_once`,

Definitely. I don't know why vim.notify_once was added...